Python/bltinmodule.c

```diff
@@ -422,6 +422,29 @@ builtin_callable(PyObject *module, PyObject *obj)
     return PyBool_FromLong((long)PyCallable_Check(obj));
 }
 
+static PyObject *
+builtin_breakpoint(PyObject *self, PyObject *args, PyObject *keywords)
+{
+    PyObject *hook = PySys_GetObject("breakpointhook");
+
+    if (hook == NULL) {
+        PyErr_SetString(PyExc_RuntimeError, "lost sys.breakpointhook");
+        return NULL;
+    }
+    Py_INCREF(hook);
+    PyObject *retval = PyObject_Call(hook, args, keywords);
+    Py_DECREF(hook);
+    return retval;
+}
+
+PyDoc_STRVAR(breakpoint_doc,
+"breakpoint(*args, **kws)\n\
+\n\
+Call the sys.breakpointhook() function, passing *args and **kws\n\
+through unchanged.  sys.breakpointhook() must accept the arguments\n\
```

Comment:
Why not simply "Call sys.breakpointhook(*args, **kwds)."? It's just a suggestion. I don't know what is the best.